File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.java
package ar.com.wolox.android.example.ui.home;

import android.content.Context;
import android.content.Intent;

import ar.com.wolox.android.R;
import ar.com.wolox.wolmo.core.activity.WolmoActivity;

/**
 * HomeActivity class
 **/
public class HomeActivity extends WolmoActivity {

    @Override
    protected void init() {
        replaceFragment(R.id.vActivityBaseContent, new HomeFragment());
    }

    @Override
    protected int layout() {
        return R.layout.activity_base;
    }

    public static void start(Context context) {
        Intent intent = new Intent(context, HomeActivity.class);
        context.startActivity(intent);
    }
}

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt
package ar.com.wolox.android.example.ui.home

import android.content.Context
import android.content.Intent
import ar.com.wolox.android.R
import ar.com.wolox.wolmo.core.activity.WolmoActivity

/**
 * HomeActivity class
 */
class HomeActivity : WolmoActivity() {
    override fun init() {
        replaceFragment(R.id.vActivityBaseContent, HomeFragment.newInstance())
    }

    override fun layout(): Int = R.layout.activity_base

    companion object {
        fun start(context: Context) {
            val intent = Intent(context, HomeActivity::class.java)
            context.startActivity(intent)
        }
    }
}